server/jest.config.js

```diff
@@ -1,8 +1 @@
-module.exports = {
-    globals: {
-        sequelize: {
-            define: obj => obj,
-            query: () => [[{ at_name_id: 1, at_name: 'Foo', active: true }]]
-        }
-    }
-};
+module.exports = {};
```

Comment:
do we need this file anymore, @evmiguel ?

Reply:
Not anymore!